parser.go

```diff
@@ -552,10 +555,16 @@ func (p *parser) parseType() (Type, error) {
 			if err != nil {
 				return nil, err
 			}
-			scale, err := strconv.Atoi(sScale)
-			if err != nil {
-				// unexpected since sIndex should conform to pIndex
-				panic(err)
+			var scale int = 32 + 1
+			if len(sScale) <= len("32") {
+				scale, err = strconv.Atoi(sScale)
+				if err != nil {
+					// unexpected since sIndex should conform to pIndex
```

Comment:
Should this comment be about scales instead of indexes? Looks like maybe just bad copypasta from above. :-)

Reply:
yes, fixing